apps/prairielearn/src/lib/editors.ts

```diff
@@ -103,17 +103,24 @@ export function getNamesForAdd(
   longNames: string[],
   shortName = 'New',
   longName = 'New',
```

Comment:
I was confused by this code at first:

This is essentially 'conflict resolution' code.

- function could be renamed `getUniqueNames`.
- ignoreShortNameCase could be flipped to `checkShortNameCase` - positive booleans are always easier to read.